node/src/components/consensus/protocols/highway.rs

```diff
@@ -42,6 +43,14 @@ use crate::{
 /// even if eras are longer than this.
 const MAX_ENDORSEMENT_EVIDENCE_LIMIT: u64 = 10000;
 
+/// The timer for creating new units, as a validator actively participating in consensus.
+const TIMER_ID_ACTIVE_VALIDATOR: TimerId = TimerId(0);
+/// The timer for adding a vertex with a future timestamp.
+const TIMER_ID_VERTEX_WITH_FUTURE_TIMESTAMP: TimerId = TimerId(1);
```

Comment:
I think I am missing something.  Why can't we have `enum TimerId = { ActiveValidator, VertexWithFutureTimestamp }`?  This way the compiler can tell us if we haven't exhaustively handled all of the cases in our code.

Reply:
It would make the `ConsensusProtocol` trait Highway-specific.
I guess we don't care much about that anymore, though, so I could do that if you prefer.

Reply:
I don't think we should be turning it into an enum – like you said, `ConsensusProtocol` is still unaware of Highway details. Let's leave it that way.